### DISCUSSION TITLE
GoPro 12 - no Bluetooth option

### DISCUSSION THREAD

**Discussion by lexirwin** [gopro/labs#806] [General]
My GoPro 12 doesn’t allow a Bluetooth pairing when connected to an external power bank

**Comment by dnewman-gpsw** [comment]
Is this is Labs question?  As the camera does allow BLE pairing with external power.

**Comment by lexirwin** [comment]
Yes it is a Labs question - is there a code to allow Bluetooth while connected to an external power bank? I know GoPro 12 has Bluetooth but when I connect to external power the Bluetooth option isn’t activated
